hammer/timing/tempus/__init__.py

```diff
@@ -127,17 +128,22 @@ def init_design(self) -> bool:
 
         if self.hierarchical_mode.is_nonleaf_hierarchical():
             # Read ILMs.
-            for ilm in self.get_input_ilms():
+            for ilm in self.get_input_ilms(full_tree=True):
                 # Assumes that the ILM was created by Innovus (or at least the file/folder structure).
                 # TODO: support non-Innovus hierarchical (read netlists, etc.)
-                verbose_append("read_ilm -cell {module} -directory {dir}".format(dir=ilm.dir, module=ilm.module))
+                verbose_append("set_ilm -cell {module} -in_dir {dir}".format(dir=ilm.dir, module=ilm.module))
```

Comment:
Why is it now `set_ilm` followed by `read_ilm` below? Just curious.

Reply:
I don't know why Cadence changed it, but in 23.1, `read_ilm` can only be called once; it reads all ILMs declared by `set_ilm`.

Reply:
Ok. Can you do a version check on this change then?